app/models/ability.rb

```diff
@@ -8,6 +8,10 @@ class Ability
 
   # Define any customized permissions here.
   def custom_permissions
+    if current_user.admin?
+      can [:create, :show, :add_user, :remove_user, :index, :edit, :update, :destroy], Role
```

Comment:
In sufia 7 user groups dont exist anymore. Instead what you do is add a user to a role and then you can check whether a role contains a user.